### ISSUE TITLE
AI improvements

### ISSUE THREAD

**Issue by devonjones** [xws-bench/bench#7]
What I would love to see is the ability to choose between AIs for ships, so example, when I fight against palp aces, the shuttle always tries to approach to attack, but it would be nice if there was an option to assert that the strategy for that ship should be to stay out of arc, and as far from heavy hitters as it can.

Some specific AI's I could see:
1) Arc dodger who will choose to get out of arc as primary goal, with getting a shot at range one as send most important (ex list: any imperial ace)
2) Ordinance, with an aim to drop a single ship (ex list: Jumpmaster 15K)
3) Stay close to ally, bump states (ex usage: HWK with kyle or jan)
4) Flee (Ex usage: palpatine shuttle)
5) Joust (B-Wing, T70)

Could be interesting to make these into modules so people can hack on the AIs independent of the rest of the code to try and make stronger opponents

**Comment by xws-bench** [comment]
Yes, I agree, these are nice goals that could be developed indeed. I'll give a try !
